tap/api/api.go

```diff
@@ -426,7 +426,7 @@ type TcpStream interface {
 	SetProtocol(protocol *Protocol)
 	GetOrigin() Capture
 	GetProtoIdentifier() *ProtoIdentifier
-	GetReqResMatcher() RequestResponseMatcher
```

Comment:
why do we need to change this to array of matchers? 
if the exception is casting the request response matcher to the interface because it is nil can't we just add condition for the "nil" of the value before casting? 

maybe there is another reason that I don't see now and you can explain.